activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -130,27 +130,37 @@ def initialize(*addresses)
       end
 
       # Increment a cached value. This method uses the memcached incr atomic
-      # operator and can only be used on values written with the +:raw+ option.
-      # Calling it on a value not stored with +:raw+ will initialize that value
-      # to zero.
+      # operator and can only be used on values written with the +:raw+ option
+      # or initially set using a previous increment call.
```

Comment:
This part makes sense when you understand the other changes in this PR, but I think it's confusing for a first time reader - to call this method you have to have already called it??

I think it's hard to document the problem with non-`raw` values because the `Entry` class is an implementation detail that's leaking here.